porcupine/plugins/run/dialog.py

```diff
@@ -105,8 +105,14 @@ def __init__(self, file_path: Path, project_path: Path, suggestions: list[histor
             validated_callback=self.update_run_button,
         )
 
-        sub_text = "\n".join("{%s} = %s" % pair for pair in substitutions.items())
-        ttk.Label(content_frame, text=f"Substitutions:\n{sub_text}\n").pack(fill="x")
+        sub_text = "\n".join("\t•  {%s} = %s" % pair for pair in substitutions.items())
```

Comment:
Instead of `\t•  `, I think it would be better to use padding on the left side to indent the items, and display no black dots. That way it will be very easy to copy/paste the substitution text into Porcupine.